src/platforms/common/data-management/event-grouping/server-side-fingerprinting.mdx

```diff
@@ -3,31 +3,20 @@ title: "Server-side Fingerprinting"
 sidebar_order: 1
 ---
 
-Server-side fingerprinting is also configured with a config similar to
-[grouping enhancements](../grouping-enhancements/),
-but the syntax is slightly different. The matchers are the same
-but instead of flipping flags, a fingerprint is assigned that overrides the
-default grouping entirely.
+Server-side fingerprinting is also configured with a config similar to [grouping enhancements](../grouping-enhancements/), but the syntax is slightly different. The matchers are the same, but instead of flipping flags, a fingerprint is assigned, and it overrides the default grouping entirely.
```

Comment:
```suggestion
Server-side fingerprinting is also configured with a config similar to [grouping enhancements](../grouping-enhancements/), but the syntax is slightly different. The matchers are the same, but instead of flipping flags, a fingerprint is assigned and it overrides the default grouping entirely.
```